lib/features/home/view_model/home_interaction_cubit.dart

```diff
@@ -32,7 +33,17 @@ class HomeInteractionCubit extends Cubit<HomeInteractionState> {
         }
       }
     } on UserNotFoundException catch (_) {
-      // TODO handle case when user is not login
+      emit(OpenLoginBottomSheetModal());
+    }
+  }
+
+  void onTapBottomItems(BottomBarItems item) async {
+    emit(StartOnTapBottomItems());
+    bool isUserLoggedIn = await authenticationService.getIsUserLoggedIn();
```

Comment:
For book value, method name should be check or validate, or use getter here instead